.azure/azure-pipelines.ci.yml

```diff
@@ -188,6 +188,37 @@ stages:
       arch: x64
       tls: openssl
 
+#
+# Regression Tests
+#
+
+- stage: regression
+  displayName: Regression
```

Comment:
Let's name these performance tests instead of regression tests. Regression protection is just one part of the perf tests. Recording the results and tracking them over time is another.